docs/README.md

```diff
@@ -55,94 +55,12 @@ Currently, this should be used to create a block analogous to an `ImageSlider`,
-| `sliderLayoutConfig` | `Object` (described below)              | Controls slider's behavior.                             | Described below. |
-| `images`             | `Array (ImageObject)` (described below) | An array of images to be rendered as a slider.          | `null`           |
-
-#### sliderLayoutConfig object
-
-| Prop name              | Type                                                       | Description                                                                                    | Default value                        |
-| ---------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------------------ |
-| `infinite`             | `Boolean`                                                  | Controls if the slider should or should not be infinite.                                       | `true`                               |
-| `usePagination`        | `Boolean`                                                  | Toggles whether or not to use a fluid scroll for navigation, disabling the notion of a "page". | `true`                               |
-| `itemsPerPage`         | `{ destop: Number, tablet: Number, phone: Number }`        | Controls how many slides should be shown on each type of device.                               | `{ destop: 1, tablet: 1, phone: 1 }` |
-| `showNavigationArrows` | `mobileOnly`&#124;`desktopOnly`&#124;`always`&#124;`never` | Controls when should navigation arrows be rendered.                                            | `'always'`                           |
-| `showPaginationDots`   | `mobileOnly`&#124;`desktopOnly`&#124;`always`&#124;`never` | Controls when should pagination dots be rendered.                                              | `'always'`                           |
-
-
-## Styles API
-
-This app provides some CSS classes as an API for style customization.
-
-To use this CSS API, you must add the `styles` builder and create an app styling CSS file.
-
-1. Add the `styles` builder to your `manifest.json`:
-
-```json
-"builders": {
-  "styles": "1.x"
-}
-```
-
-2. Create a file called `vtex.store-image.css` inside the `styles/css` folder. Add your custom styles:
-
-```css
-.imageElement {
-  margin-top: 10px;
-}
-```
+Note that the `slider-layout` block, exported from the Slider Layout app, is given as a child of `list-context.image-list`. It is responsible for defining which images will be displayed, as well as their behavior once rendered.
 
-### CSS namespaces
+This means that the `list-context.image-list` block is in fact not rendered and therefore does not have its own props that can be configured. Instead, you should use the `slider-layout` props - check out the Configuration section in the [Slider Layout documentation](https://vtex.io/docs/app/vtex.slider-layout).
```

Comment:
The `list-context.image` block actually has two props: `images`, which is an array of `Image` (the same object described in the previous version of this documentation), and also `height`, which is a `Number` that enables the user to set a height to be applied to every image inside of the `images` array.

Reply:
This is already in the example, so no need to change anything there :)
Also, how about highlighting the fact that you can `add`, `remove` and `edit` the images that are within an image-list?
I don't know if this is obvious or not 😂

Reply:
Maybe an important notice here would be to recommend people to **always** manage those images  via Site Editor